packages/gammu/gammu.0.9.4/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "1.2"
+maintainer: "Christophe Troestler <Christophe.Troestler@umons.ac.be>"
+authors: [ "Christophe Troestler"
+           "Noémie Meunier"
+           "Pierre Hauweele"]
+tags: [ "SMS" "phone" "clib:gammu" ]
+license: "LGPL-3.0 with OCaml linking exception"
+homepage: "https://github.com/Chris00/ocaml-gammu"
+dev-repo: "https://github.com/Chris00/ocaml-gammu.git"
+bug-reports: "https://github.com/Chris00/ocaml-gammu/issues"
+doc: "https://Chris00.github.io/ocaml-gammu/doc"
+build: [
+  [ "jbuilder" "subst" ] {pinned}
+  [ "jbuilder" "build" "-p" name "-j" jobs ]
+]
+build-test: [["jbuilder" "runtest" "-p" name "-j" jobs]]
+depends: [
+  "jbuilder" {build}
+  "jbuilder"  {build & >="1.0+beta9"}
+  "configurator" {build}
+  "base-unix" {build & test}
+]
+depexts: [
+  [["ubuntu"]["libgammu-dev" "pkg-config"]]
+  [["debian"]["libgammu-dev" "pkg-config"]]
+  [["centos"]["gammu-devel"  "pkgconfig"]]
```

Comment:
looks like this is in the epl centos repository -- could you try adding `epel-release` to the depext list for centos? that will add the extra remote